py/server/tests/test_time.py

```diff
@@ -886,7 +890,53 @@ def test_parse_local_time(self):
 
         time_str = "23:59x:59"
         dt = parse_local_time(time_str, quiet=True)
-        self.assertEquals(None,dt)
+        self.assertEqual(None, dt)
+
+    # endregion
+
+    # region usage in query, replace with wrapped java method
+    def test_java_method(self):
+        import deephaven.time
+        import deephaven.time as dhtime
+        from deephaven.time import now as dhnow
+        from deephaven.time import now, MINUTE, lower_bin
+
+        system_arg = True
+        resolution_arg = "ns"
+        nrow = 1000
+        formulas = [
+            "deephaven.time.now()",
+            "dhtime.now()",
+            "dhnow()",
+            "now()",
+            "now(system_arg)",
+            "now(system_arg, resolution_arg)",
+            # Python built-in constants are recognized.
```

Comment:
should test None, if that is the case.  The python keywords should probably also be added to the column name verifier so that we don't create column names that match special python words.